scripts/mysql-schema.sql

```diff
@@ -118,8 +117,7 @@ CREATE TABLE `declared_implicits` (
   `col` int(11) NOT NULL,
   `name` text NOT NULL,
   `fqn` text NOT NULL,
-  `fqtn` text NOT NULL,
-  `type` text NOT NULL,
+  `signature` text NOT NULL,
   `kind` tinytext NOT NULL,
   `nargs` int(11) NOT NULL,
```

Comment:
I think we can also remove the nargs

Reply:
Perhaps eventually, we should move to the parameter lists and parameters tables...

Reply:
Don't we need nargs to match the implicit conversion pattern? An implicit conversion can be an implicit def with nargs == '1'

Reply:
The problem is that nargs misses information. The following will look the same, no?
```
implicit def f(that: MyObject)
implicit def f(implicit that: MyObject)
```

Reply:
No, nargs ignores implicit parameters